skype/skype-ps/skype/New-CsTeamsFeedbackPolicy.md

```diff
@@ -99,14 +99,18 @@ Accept wildcard characters: False
 ### -ReceiveSurveysMode
 Set the receiveSurveysMode parameter to enabled to allow users who are assigned the policy to receive the survey.
 
+Possible values:
+ - Enabled
+ - Disabled
+
 ```yaml
-Type: Object
+Type: String
 Parameter Sets: (All)
 Aliases:
 
 Required: False
 Position: Named
-Default value: None
+Default value: Enabled
```

Comment:
```suggestion
Default value: EnabledUserOverride
```

Reply:
![image](https://user-images.githubusercontent.com/36666927/105069275-54fd5b00-5a47-11eb-8bf7-686c45787312.png)

Did you get EnabledUserOverride in your tenant as default? not on mine...

Reply:
Yes... Awesome.

![image](https://user-images.githubusercontent.com/33589238/105073683-204ccc00-5a66-11eb-998a-57279e83fb45.png)

Reply:
Different Tenants... different results... what do you say @tseward should we put "none" as default value then?